test/8.internal.urlCache.js

```diff
@@ -0,0 +1,127 @@
+"use strict";
+var urlCache = require("../lib/internal/urlCache");
+
+var expect = require("chai").expect;
+
+
+
+describe("INTERNAL -- urlCache", function()
+{
+	describe("startRetrieving()", function()
+	{
+		it("reports that it is retrieving", function()
+		{
+			var cache = new urlCache();
+			var url = "some url";
+
+			cache.startRetrieving(url);
+
+			expect(cache.isRetrieving(url)).to.be.true;
```

Comment:
a space on each side of the contents of `expect()` because the doubling up of parenthesis so close together can be more difficult to read

same goes for similar cases below